html/exercise5/script.js

```diff
@@ -0,0 +1,127 @@
+const apiKey = "6b40f680";
+const container = document.querySelector(".container");
+const searchInput = document.querySelector("#search-input");
+const searchButton = document.querySelector("#search-button");
+const sortSelect = document.querySelector("#sort-select");
+const loadMoreButton = document.querySelector("#load-more-button");
+
+let currentPage = 1;
+let currentSearch = "";
+let currentSort = "name-asc";
+
+async function fetchMovies() {
+  const url = `https://www.omdbapi.com/?apikey=${apiKey}&s=${currentSearch}&page=${currentPage}`;
+  const response = await fetch(url);
+  const data = await response.json();
+  const movies = data.Search;
+  return sortMovies(movies);
+}
+
+function sortMovies(movies) {
+  switch (currentSort) {
+    case "name-asc":
+      return movies.sort((a, b) => a.Title.localeCompare(b.Title));
+    case "name-desc":
```

Comment:
"name-asc" and "name-desc" these should be constants defined. as this improves readability